src/main/java/io/quarkus/search/app/entity/Guide.java

```diff
@@ -50,7 +51,7 @@ public class Guide {
     @Column(length = Length.LONG)
     public String title;
 
-    @FullTextField(highlightable = Highlightable.UNIFIED)
+    @FullTextField(highlightable = Highlightable.UNIFIED, termVector = TermVector.WITH_POSITIONS_OFFSETS)
```

Comment:
Wouldn't it make sense to add position offsets on the title as well? Just in case we get HTML in titles someday...